diff --git a/meson_post_install.py b/meson_post_install.py
index 01ea4c0..7658206 100644
--- a/meson_post_install.py
+++ b/meson_post_install.py
@@ -9,7 +9,7 @@ destdir = environ.get('DESTDIR', '')
 
 if not destdir:
     print('Updating icon cache...')
-    call(['gtk-update-icon-cache', '-qtf', path.join(datadir, 'icons', 'hicolor')])
+    call(['gtk4-update-icon-cache', '-qtf', path.join(datadir, 'icons', 'hicolor')])
     print("Installing new Schemas")
     call(['glib-compile-schemas', path.join(datadir, 'glib-2.0/schemas')])
 